src/AutorestSwift/Models/Complex/ParameterType.swift

```diff
@@ -57,6 +57,10 @@ enum ParameterType: Codable {
         return common.name
     }
 
+    var swiftVariableName: String {
+        return common.swiftVariableName
+    }
```

Comment:
Would it make sense to have `swiftModelName` as well? That would make the logic a bit more declarative and ensure the names are cased appropriately. We might also put an assert in here that ensures that `none-of-these` make it through. 

I would also be okay with dropping the "swift" prefix and just calling them `variableName` and `modelName`.

Reply:
Switch to use modelName where appropriate.